### ISSUE TITLE
Switching back to DW Edit destroys page

### ISSUE THREAD

**Issue by dralli72** [turnermm/ckgedit#402]
Hello, I'm currently struggling with a strange behaviour according ckgedit. I'm not really sure when this started and it doesn't happen in every situation.

I have a dokuwiki page with this source code:

```
====== Wireless LAN ======

{{tag>wlan start}}

<nav justified="true">
  * [[produkt:wlan:start|Allgemein]]
  * [[produkt:wlan:ssid:start|SSIDs]]
  * [[produkt:wlan:auth:start|Authentisierung]]
  * [[produkt:wlan:hardware:start|Hardware]]
  * [[produkt:wlan:troubleshooting:start|Troubleshooting]]
</nav>

```

When my current editor is ckgedit, and I want to switch back to DW Edit, the page contents are destroyed and replaced by

`__false__`

In my browser's console (Firefox 52.5.3, but it also happens in IE 11) this error is logged:

```
uncaught exception: Parse Error: <a class="wikilink1 curid" data-curid="true" href="/dokuwiki/doku.php?id=produkt:wlan:start" title="produkt:wlan:start">Allgemein</a></li>
	<li class="level1"><a class="wikilink2" href="/dokuwiki/doku.php?id=produkt:wlan:ssid:start" rel="nofollow" title="produkt:wlan:ssid:start">SSIDs</a></li>
	<li class="level1"><a class="wikilink2" href="/dokuwiki/doku.php?id=produkt:wlan:auth:start" rel="nofollow" title="produkt:wlan:auth:start">Authentisierung</a></li>
	<li class="level1"><a class="wikilink1" href="/dokuwiki/doku.php?id=produkt:wlan:hardware:start" title="produkt:wlan:hardware:start">Hardware</a></li>
	<li class="level1"><a class="wikilink1" href="/dokuwiki/doku.php?id=produkt:wlan:troubleshooting:start" title="produkt:wlan:troubleshooting:start">Troubleshooting</a></li>
</ul>

<p>&lt;/nav&gt;</p>
</div>
```

First I thought this might be because of the `<nav...>...</nav>` syntax (coming from bootswrapper plugin), but even when I remove this and the `{{tag>...}}` syntax, the error still persists.

Even stranger: I've created a new page with the same content, and when I edit this page, I can switch back and forth between ckgedit and dwedit with no problems. When I look at the source code, the only difference are two additional line feeds and preceding colons at every link

```
====== Wireless LAN ======

{{tag>wlan start}}

<nav justified="true">

  * [[:produkt:wlan:start|Allgemein]]
  * [[:produkt:wlan:ssid:start|SSIDs]]
  * [[:produkt:wlan:auth:start|Authentisierung]]
  * [[:produkt:wlan:hardware:start|Hardware]]
  * [[:produkt:wlan:troubleshooting:start|Troubleshooting]]

</nav>


```

I copied the new file over the old file, but when I edit the old file the problem is still there. ACL for both files are the same.

I even tried uninstalling ckgedit and re-installed it with a freshly download installation file, but that doesn't change anything.

I'm still using  DokuWiki 2017-02-19e "Frusterick Manners", Server is an Ubuntu 16.04 LTS, PHP 7.0.28. None of the conflicting plugins according the ckgedit plugin page are installed.

I currently have no idea what might have happened. The only plugins I've updated since Greebo is available are "nspages" and "bureaucracy". I'm currently not upgrading to Greebo because I had problems with Greebo and "blog" plugin on a different DokuWiki installation.

Any ideas appreciated.

Thanks, Thorsten

**Comment by turnermm** [comment]
I've tried your example in both ff and ie and using both the dokuwiki and the bootstrap templates.  I am unable to reproduce the errors.  

Also, I've never seen a console error that looks like yours.  The console typically outputs plain text not hml.  There is possibly nother plugin that is corrupting the output.  Also, your firefox  is quite outdated.  Mine is at 59.   l also tried it out in chrome, and agan no problems.  I also noticed that the bootswrapper plugin corrupts the dokuwiki template's configuration manager display.

Since this behavior is intermittent, it is hard to test for, but I would suggest disabling all your other plugins and then re-enabling to see if one of them is the issue.

**Comment by turnermm** [comment]
Looking at the console parse error--it seems as though the opening <ul> is missing together with the first <li>.

**Comment by dralli72** [comment]
Hi Myron, thanks for your reply, I realize that this might be tricky to find. I have to find a maintenance window to disable all plugins, but I'll keep you posted.

The reason for the outdated firefox version is that in our company we're using the ESR release channel for Firefox (and I think even on that channel, our software deployment staff is not up-to-date :( )

Today we realized that another plugin behaves strangely, the "blog" plugin doesn't show the article archive anymore, so it might also be possible that our DokuWiki installation has a general problem. I already rebuilt the search index, but that didn't help.
